lib/rust/launcher-shims/README.md

```diff
@@ -1,58 +0,0 @@
-# Launcher Shims
```

Comment:
Are you sure Engine don't need this crate anymore?

Reply:
Double-checked with Radek and will restore it. It's not really used currently, but also not deprecated.